src/base/utils/fs.cpp

```diff
@@ -382,3 +383,14 @@ bool Utils::Fs::isNetworkFileSystem(const QString &path)
 #endif // Q_OS_WIN
 }
 #endif // Q_OS_HAIKU
+
+std::unique_ptr<QTemporaryFile> Utils::Fs::tempFile(const QByteArray &data)
```

Comment:
It looks weird/ugly.
Why not create QTemporaryFile where it's used? This utility doesn't seem really useful.

Reply:
Saves two lines and obscure `make_unique`

Reply:
>Saves two lines and obscure `make_unique`

Would `unique_ptr` still be useful if `QTemporaryFile` was used directly?

Reply:
At least it should be called `createTemporaryFile()`.